File: app/src/main/java/vergecurrency/vergewallet/view/ui/activity/settings/ChooseCurrencyActivity.java
package vergecurrency.vergewallet.view.ui.activity.settings;

import android.os.Bundle;
import android.widget.ListView;

import androidx.annotation.Nullable;
import androidx.lifecycle.ViewModelProviders;

import vergecurrency.vergewallet.view.base.BaseActivity;
import vergecurrency.vergewallet.R;
import vergecurrency.vergewallet.view.adapter.CurrenciesAdapter;
import vergecurrency.vergewallet.viewmodel.SelectCurrencyViewModel;

public class ChooseCurrencyActivity extends BaseActivity {


	@Override
	protected void onCreate(@Nullable Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_currency_change);

		SelectCurrencyViewModel mViewModel = ViewModelProviders.of(this).get(SelectCurrencyViewModel.class);

		ListView currenciesList = findViewById(R.id.activity_select_currency_listview);
		currenciesList.setAdapter(new CurrenciesAdapter(this, mViewModel.getCurrencies()));

	}
}

File: app/src/main/kotlin/vergecurrency/vergewallet/view/ui/activity/settings/ChooseCurrencyActivity.kt
package vergecurrency.vergewallet.view.ui.activity.settings

import android.os.Bundle
import android.widget.ListView
import androidx.lifecycle.ViewModelProviders
import androidx.recyclerview.widget.DividerItemDecoration

import vergecurrency.vergewallet.view.base.BaseActivity
import vergecurrency.vergewallet.R
import vergecurrency.vergewallet.service.model.Currency
import vergecurrency.vergewallet.service.model.PreferencesManager
import vergecurrency.vergewallet.view.adapter.CurrenciesAdapter
import vergecurrency.vergewallet.viewmodel.SelectCurrencyViewModel

class ChooseCurrencyActivity : BaseActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_currency_change)

        val mViewModel = ViewModelProviders.of(this).get(SelectCurrencyViewModel::class.java)

        val currenciesList = findViewById<ListView>(R.id.activity_select_currency_listview)
        currenciesList.adapter = CurrenciesAdapter(this, mViewModel.currencies!!)
        //currenciesList.setDivider();

    }
}
